### DISCUSSION TITLE
how to create a table from a json string

### DISCUSSION THREAD

**Discussion by owentuu** [duckdb/duckdb#13072] [Q&A]
Now,we can use read_json to read a json file to create a table。How can I create a table use a json string in my java appliction？Not a json file but a json string that define in my java code

**Comment by radi2015** [comment]
"I also encountered this problem. It's strange that DuckDB strongly supports APIs like read_json, but a more common issue is how to directly handle JSON strings."

**Comment by cmdlineluser** [comment]
@radi2015 Yes, it has been requested by quite a few people:

- https://github.com/duckdb/duckdb/discussions/9558
